dependencies.yaml

```diff
@@ -91,6 +91,10 @@ ext-mbedtls:
 everest-utils:
   git: https://github.com/EVerest/everest-utils.git
   git_tag: v0.4.2
+# linux_libnfc-nci for RFID
+libnfc-nci:
+  git: https://github.com/EVerest/linux_libnfc-nci.git
+  git_tag: 202411.1.0
```

Comment:
Just a minor observation, for consistency with the other repos that use a date based version this could be 2024.11.0 (since it's date based it's not a semantic version anyway)

Reply:
Can also have a
`cmake_condition: "EVEREST_DEPENDENCY_ENABLED_LIBNFC_NCI"`

Reply:
A find_package somewhere around here needs to be added as well: https://github.com/EVerest/everest-core/blob/5cc8b6d765e267f6181047891e786dea7b9a90e3/CMakeLists.txt#L111

Reply:
Since I don't know in which situation it would be used: Is a BUILD.libnfc_nci.bazel required?

Reply:
With this entry: https://github.com/EVerest/everest-core/blob/ab1bb79c0404a4ad6c8ee291bcd6d751680054df/module-dependencies.cmake#L65 it will only be built when you build the new module and excluded if you exclude the module